index.js

```diff
@@ -294,10 +324,22 @@ module.exports = function (schema, options) {
             doc.deletedId = params.deletedId 
         }
 
-        return updateDocumentsByQuery(this, conditions, doc, callback);
+        const saveOptions = {};
+        if (params.session) {
+            saveOptions.session = params.session;
+        }
+
```

Comment:
❓ Do you need the options check in here that you do above?
```suggestion
        if(options.validateBeforeDelete === false) {
            saveOptions.validateBeforeSave = false;
        }
        
```